styles/styles.css

```diff
@@ -306,3 +314,150 @@ main .section[data-section-status='loading'],
 main .section[data-section-status='initialized'] {
   display: none;
 }
+
+/* about-us/our-history */
+
+main .section.our-history .timeline {
+  position: relative;
+}
+
+main .section.our-history h2 {
+  text-align: center;
+  color: var(--mt-our-history-color-violet);
+  font-size: 1rem;
+  font-weight: 900;
+  text-transform: uppercase;
+  letter-spacing: 4px;
+  margin-bottom: 100px;
+}
+
+main .section.our-history .year {
+  position: relative;
+  transition: background-color 0.5s linear;
+  background-color: var(--mt-our-history-color-blue);
+  box-shadow: 8px 22px 50px 0 rgb(28 28 73 / 40%);
+  font-size: 16px;
+  color: white;
+  border-radius: 4px;
+  padding: 30px 50px 10px;
+  margin-top: 120px;
+  width: 100%;
+}
+
+main .section.our-history .year.highlight {
+  background-color: var(--mt-our-history-color-violet);
+}
+
+main .section.our-history .year h3 {
+  font-size: var(--mt-body-font-size-100);
+  color: var(--mt-global-color-white);
+}
+
+main .section.our-history .year::before {
+  display: inline-block;
+  content: '';
+  background-color: var(--mt-our-history-color-blue);
+  background-image: url('/icons/calendar.svg');
```

Comment:
better have the icon as text content `:calendar:` and then use `await decorateIcons(yourSection);`

this way, the icon sprite is leveraged.

find example usages in header.js/footer.js.